docs/conf.py

```diff
@@ -31,6 +31,7 @@
 extensions = ['sphinx.ext.autosummary',
               'sphinx.ext.doctest',
               'sphinx.ext.napoleon',
+              'sphinx.ext.imgconverter',
```

Comment:
The CI doesn't actually test this new feature (the CI didn't run at all, before, but now it doesn't build latex).